README.md

```diff
@@ -1,6 +1,7 @@
 <img src="https://img.shields.io/github/v/release/guardian/mobile-apps-api-models?label=schema%20version">
 
-[![mobile-apps-api-models Scala version support](https://index.scala-lang.org/guardian/mobile-apps-api-models/mobile-apps-api-models/latest-by-scala-version.svg?platform=jvm)](https://index.scala-lang.org/guardian/mobile-apps-api-models/mobile-apps-api-models)
+[![mobile-apps-api-models-v0 Scala version support](https://index.scala-lang.org/guardian/mobile-apps-api-models/mobile-apps-api-models-v0/latest-by-scala-version.svg?platform=jvm)](https://index.scala-lang.org/guardian/mobile-apps-api-models/mobile-apps-api-models-v0)
```

Comment:
This just fixes the Scaladex badge - it was pointing to the old artifact name (`mobile-apps-api-models`) rather than the new name (`mobile-apps-api-models-v0`).